newbank/server/NewBank.java

```diff
@@ -1,11 +1,12 @@
 package newbank.server;
 
-import java.util.HashMap;
+//importing all java utility libraries
+import java.util.*;
 
 public class NewBank {
 	
 	private static final NewBank bank = new NewBank();
-	private HashMap<String,Customer> customers;
+	private HashMap<String,Customer> customer;
```

Comment:
I think there is a typo. It should be "customers", not "customer". I got a errors when trying to run it with "customer" as the HashMap name.